### PR TITLE
8.0 port picking dispatch

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: picking_dispatch/README.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Picking dispatch
+================
+
+This module allows you to group various picking into a dispatch order
```
Comment (lepistone) [inline picking_dispatch/README.rst]: pickings

**Comment by yvaucher** [comment]
@lepistone typo fixed

File: picking_dispatch/README.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The individual orders in the wave are dependent on the criteria used to make
+the selection. To organize the sequence of orders and assignment to waves,
+consistent with routing, loading and planned departure times of shipping
+vehicles or production requirements, etc., to reduce the space required for
+shipping dock handling to assemble orders and load; and To assign staff to
+each wave and function within a wave, with the expectation that all the work
+assigned to each wave will be completed within the wave period.
+
+From there, you can perform various operation :
```
Comment (lepistone) [inline picking_dispatch/README.rst]: `operation :` -> `operations:`

**Comment by yvaucher** [comment]
@lepistone typo fixed (again)

File: picking_dispatch/report/dispatch_report.py [hunk 3]
```diff
@@ -143,7 +164,45 @@ def set_context(self, objects, data, ids, report_type=None):
         return super(PrintDispatch, self).set_context(new_objects, data, ids,
                                                       report_type=report_type)
 
-report_sxw.report_sxw('report.webkit.dispatch_order',
-                      'picking.dispatch',
-                      'addons/picking_dispatch/report/dispatch.html.mako',
-                      parser=PrintDispatch)
+#class PrintDispatch(models.AbstractModel):
```
Comment (lepistone) [inline picking_dispatch/report/dispatch_report.py]: why is this commented out? can we remove it altogether and maybe leave a note in the manifest?

**Comment by yvaucher** [comment]
cleaned report python file

**Comment by yvaucher** [comment]
Fixed a tiny pep8 error in picking_dispatch

**Comment by lepistone** [comment]
thanks! :+1:

**Comment by jgrandguillaume** [comment]
:+1:  tested and approved.

**Pull request merged**